### PR TITLE
Fix globalID instantiations (for GPU gid64 builds)

### PR DESCRIPTION
**Pull request by liu15**

### CODE REVIEW

**Review by robinson96 — APPROVED** [review]
LGTM

File: src/care/scan.cpp [hunk 1]
```diff
@@ -39,21 +39,21 @@ void getFinalScanCount(chai::ManagedArray<int> scanvar, int length, int& scanCou
    scanCount = scanvar.pick(length);
 }
 
-#if CARE_HAVE_LLNL_GLOBALID
+#if CARE_HAVE_LLNL_GLOBALID && GLOBALID_IS_64BIT
```
Comment (adayton1) [inline src/care/scan.cpp:42]: I guess I'm confused why we have this GLOBALID_IS_64BIT as part of the macro guard. Don't we want these instantiations regardless of whether GIDTYPE is 64 bits or a regular int?

Comment (liu15) [reply src/care/scan.cpp:42]: GIDTYPE is the raw type, so it is either int or int64_t.  If it is int, then this version is a duplicate and the compiler complains.

Comment (adayton1) [reply src/care/scan.cpp:42]: Ah, that makes more sense.

**Review by adayton1 — APPROVED** [review]
LGTM

**Pull request merged**